tools/run_presubmit

```diff
@@ -682,34 +682,6 @@ def CheckBannedCpp(changed_files: List[str]) -> List[str]:
   return errors
 
 
-def CheckBadCppPatterns(changed_files: List[str]) -> List[str]:
```

Comment:
I think we still want to keep this check we should just turn it into a rachet (count how many instances there are and then make sure it doesn't increase).

It's still the case we should *not* be using trigger_config in the service except for implmenting GetTriggerMode itself.

Reply:
No, this is too complicated.

Also, we can use `trigger_config()`, we're using it correctly in many places. The presubmit check was wrong to begin with. This, in my opinion is proof of the fact that stuff like this doesn't belong in presubmits.